tests/k8s/1-006_validate-managed-by-chain/04-create-application.yaml

```diff
@@ -0,0 +1,16 @@
+# This creates an application with a destination in the other managed namespace
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: test-1-12-custom
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/redhat-developer/gitops-operator
```

Comment:
Relying on something in the gitops-operator repo might be a little fragile; a change in this other repo could break this test. Maybe we should create the equivalent app here in the argocd-operator repo.